.github/replies.yml

```diff
@@ -1,4 +1,7 @@
 replies:
+  - body: |
+      👋 Hey @{{ author }}! Just checking in, is this still something you have time for? No worries if not - I just don't want to leave it hanging.
```

Comment:
This just reminded me of https://github.com/JoshuaKGoldberg/refined-saved-replies/issues/45...